src/modules/frost/main_impl.h

```diff
@@ -129,7 +128,9 @@ static SECP256K1_WARN_UNUSED_RESULT int deserialize_frost_signature(secp256k1_fr
                                                                     const unsigned char *serialized_signature) {
     secp256k1_fe x;
     secp256k1_ge deserialized_point;
-    secp256k1_fe_set_b32(&x, serialized_signature);
+    if (secp256k1_fe_set_b32_limit(&x, serialized_signature) == 0){
+        return 0;
+    }
```

Comment:
The migration from `secp256k1_fe_set_b32()` to `secp256k1_fe_set_b32_limit()` is ok as part of the update to 0.3.2 (other modules did the same).

However, we are also adding an additional error checking here. That modification should be done as a separate commit, before moving to 0.3.2 imho.

Reply:
Done in v2 (force pushed)